torch/_torch_docs.py

```diff
@@ -7951,10 +7951,17 @@ def merge_dicts(*dicts):
 dimension :attr:`dim` is input from :attr:`start` to ``start + length``. The
 returned tensor and :attr:`input` tensor share the same underlying storage.
 
+:attr:`start` accepts a `Tensor` for models that have it as such before calling
```

Comment:
I would delete this paragraph and just note that `start` can be a tensor below (which this PR already does)